source/process/docs-reviewers-guide.rst

```diff
@@ -0,0 +1,116 @@
+4. Once merged, PRs are verified by a Project Member and the submitter's work acknowledged. 
+5. Reviewers with consistent performance are acknowledge with an increase in skill level. Reviewers with repeated inconsistency may have their level decreased (though this is rare). 
+
+The following sections lay out the details for this process: 
+
+Pre-reading 
+-------------
+
+- Please read `Contributor's Guide for docs.mattermost.com XXXXXX <XXXX>`_ for the process by which Approved Contributors submit PRs and engage with reviewers. 
+
+Review Labels 
+-------------
+
+The following labels are used to indicate the type of review required for each doc PR: 
+
+- **Needs Style Guide Approval** - Review for `style guide requirements <https://docs.mattermost.com/process/sg_mattermost-doc-style.html>`_, and specifically: 
+
+   - Verify all links work
+   - Verify grammar is correct based on US English requirements 
+   - Verify RST renders properly in GitHub preview
+
+- **Needs Technical Approval** - Technical change requiring developer review, e.g. change in system requirements.
+
+- **Needs Testing Approval** - Change requires manual testing, e.g. significant install process change. 
+
+- **Needs Sphinx Approval** - Significant structural changes to CSS or Sphinx features, requires HTML generation and verification. 
+
+- **Needs Structural Approval** - Approval required on changes to TOC or organization.
+
+- **Needs Verification & Acknowledgement** - docs.mattermost.com needs to be verified and acknowledgement added to PR.
+
+- **Post-Merge Editing Required** - PR can be merged and someone will follow-up with edits for minor corrections.
+
+- **Awaiting Submitter Action** - Merge should not happen until feedback is addressed. Note: This label should be applied for any work-in-progress PR, a separate label is not needed. 
+
+Levels 
+-------------
+
+The anticipated consistency of reviews by Reviewers is noted using the following levels: 
+
+- **Level 1** - Self-identified expertise in a review criteria, and willing to help.
+- **Level 2** - 80%+ track record of good reviews, or subjective promotion to this level by Level 3 or above. 
+- **Level 3** - 90%+ track record of correct reviews, or subjective promotion to this level by Level 4 or above. 
+- **Level 4** - 95%+ track record of correct reviews or maintainer of doc repo.
+
+Extended periods of consistent performance are recognized by level increases. Repeated inconsistency reduces an reviwers level. 
+
+Roles
+-------------
+
+The following lists roles involved in the PR review process: 
+
+- **Approved Contributor** - Anyone who's completed the CLA. This is the minimum role to submit a PR for review. 
+- **Project Member** - Approved Contributor granted ability to apply labels. Often a staff member from Mattermost, Inc. 
+- **Project Member Reviewer on call** - Member assigned to review PRs in certain approval categories. 
```

Comment:
Why is it "on call", are people assigned this role over time periods? It's not very clear.